packages/models-library/tests/test_service_resources.py

```diff
@@ -0,0 +1,57 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from pprint import pformat
+from typing import Type
+
+import pytest
+from models_library.services_resources import (
+    ComposeImage,
+    ImageResources,
+    Resources,
+    ServiceResources,
+)
+from pydantic import BaseModel, parse_obj_as
+
+
+@pytest.mark.parametrize(
+    "model_cls",
+    (
```

Comment:
There is no need to do this type of tests anymore. 

Since last week we have a parametrize tests that does this with all ``BaseModel`` subclasses defined in the ``models_library`` module. 

See ``packages/models-library/tests/test__models_examples.py``

Reply:
Great! Apparently I've forgotten about it. Removed this.